### ISSUE TITLE
IWD2: Mysterious bonus in mage spell slots 

### ISSUE THREAD

**Issue by MarcelHB** [gemrb/gemrb#680]
#### Bug description
I have a mage, WIS 10, INT 17, level 1. I was given then choice to pick 2 spells in IWD2's native CharGen. In the spellbook, GemRB gives me another slot that is not available in IWD2.

GemRB version: master

#### Steps to reproduce
1. Create an IWD2 char as mentioned above.

#### Expected behavior
A total of 2 mage spell slots.

#### Actual behaviour
I get 3 spell slots. The reason is that `CRESpellMemorization::SlotCount == 2` while `CRESpellMemorization::SlotCountWithBonus == 3` (that is returned in `GUIScripts/iwd2/GUISPL.py:188`). Not sure where is this coming from exactly.

**Comment by lynxlynxlynx** [comment]
Should be from your high int (mxsplbon.2da), but it seems wizards are excluded? Odd.

In other games, this was there only for priests (mxwisbon.2da).

**Comment by lynxlynxlynx** [comment]
I've created a fresh human wizard with 17 int and in both the original and gemrb I get 2 memorization slots. The original did let me learn 3 spells during chargen instead of 2 though.

**Comment by lynxlynxlynx** [comment]
I've fixed the bonus thing, so in my experience nothing is wrong any more with the numbers, comparing character generation.

Or was your report about loading a character created in the original into gemrb and having inconsistent slots?

**Comment by MarcelHB** [comment]
Should have mentioned that, yes, loading a native savegame into GemRB brought me here.

**Comment by lynxlynxlynx** [comment]
Yeah, it's mxsplbon. Currently things almost work out, since only one memo is reset, not saved through chargen. Fixing that creates the same problem visible by loading an original game.

So the challenge is in reconciling the two. We have to take it into account already in chargen, so users can memorize everything, but then once the clock starts ticking, we shouldn't be granting the bonus twice. Maybe moving that bit of code to the stat pcfs would do and then the manual bonus granting from chargen can be removed.
